src/Commands/MigrationDiffVendorCommand.php

```
@@ -0,0 +1,157 @@
+                continue;
+            }
+
+            $target = $localMap[$strippedName];
+
+            $result = $this->compareFileContents($vendorFile, $target, $fs);
+
+            if ($result['status'] === 'unchanged') {
+                $unchanged[] = $target;
+            } else {
+                $modified[] = ['path' => $target, 'diff' => $result['diff']];
+            }
```

PHPMD (2.15.0)

The method collectAndCategorizeFiles uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)